run-tests-with-build-optawebs-parent/optawebs-project-sources-test/optaweb-employee-rostering-test-script-openshift.sh

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env bash
+
+this_script_directory="${BASH_SOURCE%/*}"
+if [[ ! -d "${this_script_directory}" ]]
+then
+  this_script_directory="$PWD"
+fi
+
+# shellcheck source=common.sh
+. "${this_script_directory}/common.sh"
+
+function display_help() {
+  readonly script_name="./$(basename "$0")"
+
+  echo "This script tests deployment of Optaweb Employee Rostering on OpenShift."
+  echo "Usage:"
+  echo "  ${script_name} PROJECT_BASEDIR CYPRESS_IMAGE_VERSION OPENSHIFT_API_URL OPENSHIFT_USER OPENSHIFT_PASSWORD"
+  echo "  ${script_name} --help"
+}
+
+if [[ $1 == "--help" ]]
+then
+  display_help
+  exit 0
+fi
+
+readonly project_basedir=$1
+[[ -d ${project_basedir} ]] || {
+  echo "Project base directory $project_basedir does not exist!"
+  display_help
+  exit 1
+}
+
+# login to OpenShift
+readonly openshift_api_url=$3
+readonly openshift_user=$4
+readonly openshift_password=$5
+readonly settings_file=$6
+
+oc login -u "${openshift_user}" -p "${openshift_password}" "${openshift_api_url}" --insecure-skip-tls-verify=true
+
+#create new empty project
+readonly uuid=$(uuidgen)
+readonly openshift_project=employee-rostering-${uuid:0:8}
+oc new-project "${openshift_project}"
+
+# make sure the project is ready
+oc get project "${openshift_project}"
+
+chmod u+x "${project_basedir}"/runOnOpenShift.sh
+
+sed 's#mvn clean install -DskipTests -Dquarkus.profile=postgres#mvn clean install -DskipTests -Dquarkus.profile=postgres -Denforcer.skip -s '$settings_file'#g' "${project_basedir}"/runOnOpenShift.sh
```

Comment:
Wouldn't it be good to also alter the orginal shell script to allow passing settings.xml file and turning off the enforcer? Though just a hint for a possible follow-up, I understand it's kinda late for this release.

Reply:
That is a good idea for the next release 
https://issues.redhat.com/browse/PLANNER-2432